pkg/karmadactl/token/token.go

```diff
@@ -300,11 +300,11 @@ func (o *CommandTokenOptions) runDeleteTokens(out io.Writer, client kubeclient.I
 		}
 
 		tokenSecretName := bootstraputil.BootstrapTokenSecretName(tokenID)
-		klog.V(1).Infof("[token] deleting token %q", tokenID)
+		klog.V(1).Infof("[token] deleting secret %s", tokenSecretName)
 		if err := client.CoreV1().Secrets(metav1.NamespaceSystem).Delete(context.TODO(), tokenSecretName, metav1.DeleteOptions{}); err != nil {
-			return fmt.Errorf("failed to delete bootstrap token %q, err: %w", tokenID, err)
+			return fmt.Errorf("failed to delete secret %s, err: %w", tokenSecretName, err)
 		}
-		fmt.Fprintf(out, "bootstrap token %q deleted\n", tokenID)
+		fmt.Fprintf(out, "bootstrap token deleted\n")
```

Comment:
```suggestion
		fmt.Fprintf(out, "bootstrap token %s deleted\n", tokenSecretName)
```
Printing out the `tokenSecretName` may help in concurrency scenarios for problem localization

Reply:
no deed to print out `tokenSecretName`.

Pay attention to this is `fmt.Fprintf`, not `klog.Infof`, it will not print in log file.
Actully, this is a redundantinformation, which has nothing to with `problem localization`.

```console
$ karmadactl token create
idrqgm.qtp3aqzjya9y5cc3
$ karmadactl token delete idrqgm.qtp3aqzjya9y5cc3                       
bootstrap token "idrqgm" deleted
```

this print actually is a immediate reponse to `karmadactl token delete`, since above line has specified token, we just need reponse it `bootstrap token deleted`, which represents delete action did success.

Reply:
As you can see, command `karmadactl token [delete]` will delete a list of bootstrap tokens, so, printing out `tokenSecretName` may be useful when the token list has more than one token~
```bash
(base) ➜  karmada git:(main) ✗ karmadactl token create                     
1w722y.cvjqi00xtih7kpb6
(base) ➜  karmada git:(main) ✗ karmadactl token create
164ceb.erp1k8aqwa8gc1sn
(base) ➜  karmada git:(main) ✗ karmadactl token delete 1w722y 23hjss 164ceb
bootstrap token "1w722y" deleted
error: failed to delete bootstrap token "23hjss", err: secrets "bootstrap-token-23hjss" not found
```

Reply:
since this line is not print to log file, I prefer just keep previous `fmt.Fprintf(out, "bootstrap token %q deleted\n", tokenID)` unchanged.